beacon-chain/rpc/eth/light-client/handlers_test.go

```diff
@@ -42,427 +42,221 @@ func TestLightClientHandler_GetLightClientBootstrap(t *testing.T) {
 	cfg.FuluForkEpoch = 5
 	params.OverrideBeaconConfig(cfg)
 
-	t.Run("altair", func(t *testing.T) {
-		l := util.NewTestLightClient(t, version.Altair)
+	versionToForkEpoch := map[int]primitives.Epoch{
+		version.Altair:    params.BeaconConfig().AltairForkEpoch,
+		version.Bellatrix: params.BeaconConfig().BellatrixForkEpoch,
+		version.Capella:   params.BeaconConfig().CapellaForkEpoch,
+		version.Deneb:     params.BeaconConfig().DenebForkEpoch,
+		version.Electra:   params.BeaconConfig().ElectraForkEpoch,
+	}
 
-		slot := primitives.Slot(params.BeaconConfig().AltairForkEpoch * primitives.Epoch(params.BeaconConfig().SlotsPerEpoch)).Add(1)
-		blockRoot, err := l.Block.Block().HashTreeRoot()
-		require.NoError(t, err)
+	for testVersion := 1; testVersion <= 5; testVersion++ {
```

Comment:
To avoid magic numbers and make this more future-proof, you can write
```suggestion
	for testVersion := version.Altair; testVersion <= len(versionToForkEpoch); testVersion++ {
```